packages/components/src/navigable-container/container.tsx

```diff
@@ -98,9 +119,9 @@ class NavigableContainer extends Component {
 			// from scrolling. The preventDefault also prevents Voiceover from
 			// 'handling' the event, as voiceover will try to use arrow keys
 			// for highlighting text.
-			const targetRole = event.target.getAttribute( 'role' );
+			const targetRole = ( event.target as HTMLDivElement ).getAttribute( 'role' );
 			const targetHasMenuItemRole =
-				MENU_ITEM_ROLES.includes( targetRole );
+				targetRole && MENU_ITEM_ROLES.includes( targetRole );
```

Comment:
Adding `targetRole` prevents the TS error:

```sh
  Type 'null' is not assignable to type 'string'.ts(2345)
```

Reply:
Maybe we could force `targetRole` to a boolean, so that the type of `targetHasMenuItemRole` keeps being a boolean too?

```suggestion
				!! targetRole && MENU_ITEM_ROLES.includes( targetRole );
```

Reply:
Good idea, committed in https://github.com/WordPress/gutenberg/pull/49377/commits/0c8173709611342184db3565910fdcdcfae438ca